lib/quby/questionnaires/entities/questions/date_question.rb

```diff
@@ -3,57 +3,79 @@ module Questionnaires
     module Entities
       module Questions
         class DateQuestion < Question
+          POSSIBLE_COMPONENTS = %i( day month year hour minute )
+          COMPONENT_KEYS = Hash[POSSIBLE_COMPONENTS.zip %w( dd mm yyyy hh ii)]
+          COMPONENT_PLACEHOLDERS = Hash[POSSIBLE_COMPONENTS.zip %w( DD MM YYYY hh mm)]
```

Comment:
Ik denk dat het (zeker bij Quby2) de moeite waard is om een datumklasse met ondersteuning voor gedeeltelijke datums te maken, dan zouden we eenvoudig ondersteuning voor `I18n.l` kunnen toevoegen denk ik.